scripts/update_envoy.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Update the Envoy SHA in istio/proxy WORKSPACE with the first argument
```

Comment:
What does `first argument` mean here?

Reply:
The following logic suggests the first argument is envoy commit sha. You can just get envoy SHA in this script I think? i.e. move SHA resolving logic from automator to here.

Reply:
not really? ENVOY SHA get from automator https://github.com/istio/test-infra/blob/master/tools/automator/automator.sh#L183
and was passed to the postsubmit job via: https://github.com/istio/test-infra/pull/2938/files#diff-f76728f0ac7a053f589864f27567f6c5R46

Reply:
> What does `first argument` mean here?
i can update the comment to clarify this is the ENVOY SHA